bliss/metrics.py

```diff
@@ -43,10 +43,19 @@ class BlissMetrics(Metric):
     star_tp: Tensor
     star_fp: Tensor
 
-    # classification metrics
-    star_flux: List
-    disk_flux: List
-    bulge_flux: List
+    # Classification metrics
+    disk_frac: List
+    bulge_frac: List
+    u_flux_s: List
+    g_flux_s: List
+    r_flux_s: List
+    i_flux_s: List
+    z_flux_s: List
```

Comment:
Are these being computed somewhere? Unless I'm just missing it, it looks like the old star flux calculation was removed but the new ones haven't been added.

Reply:
I had to take it out because of some failed pre-commit checks. Currently adding them back!